README.md

```diff
@@ -6,4 +6,100 @@
 </p>
 
 # index
-Repo housing contracts, deploys, and js library for INDEX
+
+This repo houses the [index-coop][22]'s Solidity smart contracts.
+
+[22]: https://www.indexcoop.com/
+
+## Install (for development)
+
+```
+yarn
+```
+
+### Run Hardhat EVM
+
+`yarn chain`
+
+### Build Contracts
+
+`yarn compile`
+
+### Generate TypeChain Typings
+
+`yarn build`
+
+### Run Contract Tests
+
+`yarn test` to run compiled contracts
+
+OR `yarn test:clean` if contracts have been typings need to be updated
+
+### Run Coverage Report for Tests
+
+`yarn coverage`
+
+## Installing from `npm`
+
+`index-coop` publishes its contracts as well as [hardhat][22] and [typechain][23] compilation
+artifacts to npm.
+
+The distribution comes with fixtures for mocking and testing interactions with other protocols
+including Uniswap and Compound. To use these you'll need to install the peer dependencies listed in `package.json`.
+
+```
+npm install @setprotocol/index-coop-contracts
+```
+
+[22]: https://www.npmjs.com/package/hardhat
+[23]: https://www.npmjs.com/package/typechain
+
+## Contributing
+We highly encourage participation from the community to help shape the development of Index-Coop. If you are interested in developing on `index-coop` or have any questions, please ping us on [Telegram](https://t.me/joinchat/Fx8D6wyprLUlM1jMVnaRdg).
+
+## Security
+
+### TODO: Independent Audits
+
+### Code Coverage
+
+All smart contracts are tested and have 100% line and branch coverage.
+
+### Vulnerability Disclosure Policy
```

Comment:
Does index-coop have it's own vuln disclosure protocol?

Reply:
Same as Set's